napari/_qt/layer_controls/qt_surface_controls.py

```diff
@@ -41,6 +52,18 @@ def __init__(self, layer):
         self.grid_layout.addLayout(colormap_layout, 3, 1)
         self.grid_layout.addWidget(QLabel(trans._('blending:')), 4, 0)
         self.grid_layout.addWidget(self.blendComboBox, 4, 1)
+        self.grid_layout.addWidget(QLabel(trans._('shading:')), 5, 0)
+        self.grid_layout.addWidget(self.shadingComboBox, 5, 1)
+
         self.grid_layout.setRowStretch(5, 1)
```

Comment:
```suggestion
        self.grid_layout.setRowStretch(6, 1)
```
@alisterburt I wonder if the stretch issue will be fixed by making this `6`, you might want to check first before committing, but then i think this is done!